src/FormControl/FormControl.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import isUndefined from 'lodash/isUndefined';
+import { isUndefined, get as lodashGet, set as lodashSet } from 'lodash';
```

Comment:
Import lodash on demand to avoid excessive package size in projects that do not use Tree shaking.

```suggestion
import isUndefined from 'lodash/isUndefined';
import lodashGet from 'lodash/lodashGet';
import lodashSet from 'lodash/lodashSet';
```

Reply:
> Import lodash on demand to avoid excessive package size in projects that do not use Tree shaking.

typo error:
`lodash/lodashGet` should fix to `lodash/get`，and lodashSet too

Reply:
Yes, it should be `lodash/get`